packages/analytics-js-plugins/__tests__/customConsentManager/index.test.ts

```
@@ -3,6 +3,7 @@ import { defaultLogger } from '@rudderstack/analytics-js-common/__mocks__/Logger
 import { defaultErrorHandler } from '@rudderstack/analytics-js-common/__mocks__/ErrorHandler';
 import { resetState, state } from '../../__mocks__/state';
 import { CustomConsentManager } from '../../src/customConsentManager';
+import { defaultLogger } from '../../__mocks__/Logger';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'defaultLogger'. Consider to delete it or rename it.

'defaultLogger' is defined here:


(lint/suspicious/noRedeclare)